src/lib/math/pcurves/pcurves_impl.h

```diff
@@ -831,16 +831,16 @@ class ProjectiveCurvePoint {
 
       constexpr const FieldElement& z() const { return m_z; }
 
-      constexpr void ct_poison() const {
-         x().ct_poison();
-         y().ct_poison();
-         z().ct_poison();
+      constexpr void _const_time_poison() const {
+         CT::poison(m_x);
+         CT::poison(m_y);
+         CT::poison(m_z);
```

Comment:
For short, `CT::poison_all(m_x, m_y, m_z)` would also work. Same for unpoison, of course.